packages/dashboard-core-plugins/src/panels/ConsolePanel.jsx

```diff
@@ -302,6 +303,18 @@ export class ConsolePanel extends PureComponent {
                 <>
                   <div>&nbsp;</div>
                   <div>{ConsoleConstants.LANGUAGE_MAP.get(language)}</div>
+                  <div>&nbsp;</div>
+                  <div>
+                    <SizeUsageUI
+                      connection={connection}
+                      UIParams={{
+                        defaultUpdateInterval: 10000,
+                        hoverUpdateInterval: 3000,
+                      }}
+                      monitorDuration={60 * 10}
```

Comment:
Now I wonder if 10 minutes is too long, graph moves so slow. Maybe 5?

Thoughts?

Reply:
Windows task manager shows performance over 60s, so maybe we move to 5 minutes and double the poll rate?